products/jbrowse-desktop/src/JBrowse.tsx

```diff
@@ -12,6 +13,10 @@ import { RootModel } from './rootModel'
 const JBrowse = observer(
   ({ pluginManager }: { pluginManager: PluginManager }) => {
     const { rootModel } = pluginManager
+    // @ts-ignore
+    window.root = rootModel
+    // @ts-ignore
+    window.runInAction = runInAction
```

Comment:
we can probably remove these, IIRC it was for our debugging session

Reply:
yeah, I forgot to remove those. Removed.